### DISCUSSION TITLE
Big doors

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4124] [Feature Suggestions]
This doors are seen in CO-OP raven ending.

With input they opens.

**Comment by FrozenDragon0** [comment]
I don't see the purpose of this door in the puzzlemaker, and considering it's size it's really something you should be using hammer for.

**Comment by slambo311** [comment]
1. It's a crow, not a raven.
2. Like FrozenDragon0 said, this is more of a hammer thing than a puzzlemaker thing.
3. There are similar doors in singleplayer.

**Reply by vrad-exe** [reply]
Please stop commenting on every single post, you've added nothing useful to this discussion

**Reply by slambo311** [reply]
Alright.
